src/core/public/chrome/README.md

```diff
@@ -112,6 +112,31 @@ Gets an Observable of the array of recently accessed history :-
    chrome.docTitle.change('My application title')
    chrome.docTitle.change(['My application', 'My section'])
    ```
+### GroupService:
+- Interface : ChromeNavGroup
+- Methods :
+Register a group :-
+
+`addNavLinksToGroup: (group: ChromeNavGroup, navLinks: ChromeRegistrationNavLink[]) => void;`
+
+Gets an Observable of the array of registered groups :-
+
+`getNavGroupsMap$: Observable<Record<string, NavGroupItemInMap>>`
+##### Register a new group with a navLink
+
+  ```ts
+    chrome.addNavLinksToGroup(
```

Comment:
This doesn't follow the pattern set in `chrome` where each service has its own accessor property.

Reply:
I see, will adjust the doc.

Reply:
Changed, move the example to the comment of interface.